source/backend/api/Services/LeaseService.cs

```diff
@@ -88,6 +91,11 @@ public PimsLease GetById(long leaseId)
                     property.Location = GeometryHelper.CreatePoint(newCoords, SpatialReference.WGS84);
                 }
             }
+
+            var checkListItems = lease.PimsLeaseChecklistItems.ToList();
```

Comment:
why is this necessary?

Reply:
I couldn't pass a property as a ref to a method, I get C# warn.

Reply:
oh, I meant why are we requesting/appending checklist items in the lease get method. Shouldn't that be done in a separate call to get the checklist items?
so on the frontend:
1) get the lease
2) get the checklist items for the lease.
3) combine 1 and 2.

Reply:
see existing pattern in useLeaseDetail for an example.

Reply:
Updated